src/hooks/useMarketoForm.js

```diff
@@ -2,7 +2,18 @@ import { useEffect } from 'react';
 
 const useMarketoForm = (munchkinId, id, publishableKey) => {
   useEffect(() => {
-    window.MktoForms2.loadForm('//app-abj.marketo.com', munchkinId, id);
+    window.MktoForms2.loadForm(
+      '//app-abj.marketo.com',
+      munchkinId,
+      id,
+      (form) => {
+        // eslint-disable-next-line no-unused-vars
+        form.onSuccess(function (values, followUpUrl) {
+          location.href = 'https://developer.newrelic.com/nerd-days-thank-you';
+          return false;
```

Comment:
I'm not sure if the `return false` is needed here! Could you explain the reasoning behind this?

Reply:
Removing the return statement